src/vaadin-scroll-container.html

```diff
@@ -0,0 +1,75 @@
+<!--
+@license
+Copyright (c) 2017 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<link rel="import" href="../../polymer/polymer-element.html">
+<link rel="import" href="../../vaadin-themable-mixin/vaadin-themable-mixin.html">
+<link rel="import" href="../../vaadin-element-mixin/vaadin-element-mixin.html">
+
+<dom-module id="vaadin-scroll-container">
+  <template>
+    <style>
+      :host {
+        display: block;
+        overflow: auto;
+      }
+
+      :host([horizontal-scroll-disabled]) {
+        overflow-x: hidden;
+      }
+
+      :host([vertical-scroll-disabled]) {
+        overflow-y: hidden;
+      }
+
+    </style>
+
+    <slot></slot>
+
+  </template>
+
+  <script>
+    (function() {
+      /**
+       * `<vaadin-scroll-container>` provides a simple way to display scrollbars when its content is overflowing.
+       *
+       * ```
+       * <vaadin-scroll-container>
+       *   <div>Content</div>
+       * </vaadin-scroll-container>
+       * ```
+       * ### Styling
+       *
+       * The following state attributes are available for styling:
+       *
+       * Attribute    | Description
+       * -------------|-------------
+       * `horizontal-scroll-disabled` | Set to disable horizontal scrolling
+       * `vertical-scroll-disabled` | Set to disable vertical scrolling
+       *
+       * @memberof Vaadin
+       * @mixes Vaadin.ThemableMixin
```

Comment:
Do we really need `Vaadin.ThemableMixin` here? Is it needed for element that only has one `slot`?
We don't have any shadow parts, and both the element and its content can be styled from outside.

Not a blocker if this has been already decided, just curious.

Reply:
Themable mixin still allows you to create style modules (with :host selector) that affect all scroll-containers on the page.
I would also keep it for consistency with other components.
But we can discuss it with the team if needed.